templates/common/render/manifest.go

```diff
@@ -112,19 +113,9 @@ func writeManifest(ctx context.Context, p *writeManifestParams) (rErr error) {
 		rErr = errors.Join(rErr, fh.Close())
 	}()
 
-	header := map[string]string{
-		"api_version": decode.LatestAPIVersion,
-		"kind":        decode.KindManifest,
-	}
-	headerYAML, err := yaml.Marshal(header)
-	if err != nil {
-		return fmt.Errorf("internal error: failed marshaling api_version: %w", err)
-	}
-
 	buf = append(
 		[]byte("# Generated by the \"abc templates\" command. Do not modify.\n"),
-		append(headerYAML,
-			buf...)...)
+		buf...)
```

Comment:
Is it possible to have a manifest just with header? I guess no? I'm asking this for whether we need to add a test case with just header avaiailable?

Reply:
I don't think so, there are several required fields in the manifest.